app/uk/gov/hmrc/nisp/services/StatePensionService.scala

```diff
@@ -53,6 +55,11 @@ class StatePensionService @Inject()(statePensionConnector: StatePensionConnector
           Left(StatePensionExclusionFiltered(Exclusion.Dead))
         case ex: Upstream4xxResponse if ex.upstreamResponseCode == FORBIDDEN && ex.message.contains(exclusionCodeManualCorrespondence) =>
           Left(StatePensionExclusionFiltered(Exclusion.ManualCorrespondenceIndicator))
+        case ex: Upstream4xxResponse if ex.upstreamResponseCode == FORBIDDEN && ex.message.contains(exclusionCodeCopeProcessingFailed) =>
+          Left(StatePensionExclusionFiltered(Exclusion.CopeProcessingFailed))
+        case ex: Upstream4xxResponse if ex.upstreamResponseCode == FORBIDDEN && ex.message.contains(exclusionCodeCopeProcessing) =>
+          Left(StatePensionExclusionFilteredWithCopeDate(Exclusion.CopeProcessing, copeAvailableDate = getDateWithRegex(ex.message)))
+        // Case match not exhaustive
```

Comment:
It's fine that this case match isn't exhaustive as any other exceptions will be propagated up which is what we want